### ISSUE TITLE
UNet Benchmark problem

### ISSUE THREAD

**Issue by calebchencs7** [ChenHongruixuan/BRIGHT#25]
I use your code at windows system, with 2 NVIDIA RTX 4090 GOU, and my training settings are below:
cmd = [
    "python",
    "bda_benchmark/script/standard_ML/train_UNet.py",
    
    "--dataset", "BRIGHT",
    "--train_batch_size", "8",
    "--eval_batch_size", "4",
    "--num_workers", "4", 
    "--crop_size", "640", 
    "--max_iters", "12000", 
    "--learning_rate", "1e-4", 
    "--model_type", "UNet",
    "--model_param_path", CKPT_PATH,
    "--train_dataset_path", DATA_PATH,
    "--train_data_list_path", "./bda_benchmark/dataset/splitname/standard_ML/train_set.txt",
    "--val_dataset_path", DATA_PATH,
    "--val_data_list_path", "./bda_benchmark/dataset/splitname/standard_ML/val_set.txt",
    "--test_dataset_path", DATA_PATH,
    "--test_data_list_path", "./bda_benchmark/dataset/splitname/standard_ML/test_set.txt"
]
And the result is 
The accuracy of the best round is  {'best iter': 1500, 'loc f1 (val)': 76.82089515164088, 'clf f1 (val)': 0.0, 'OA (val)': 91.77199504414543, 'mIoU (val)': 40.18765780120896, 'sub class IoU (val)': array([92.27987694, 57.40505773,  0.        , 11.06569654]), 'loc f1 (test)': 77.4457750443877, 'clf f1 (test)': 0.0, 'OA (test)': 92.51855415114372, 'mIoU (test)': 41.07613634341678, 'sub class IoU (test)': array([92.83264553, 58.88091159,  0.        , 12.59098825])}

Why the third subclass loU is 0?
